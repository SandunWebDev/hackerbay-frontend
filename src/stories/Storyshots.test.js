// This enable all stories as Jest snapshot testing cases.

import initStoryshots from "@storybook/addon-storyshots";

initStoryshots();
