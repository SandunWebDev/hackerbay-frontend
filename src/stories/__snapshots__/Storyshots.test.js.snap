// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots App Intitial View 1`] = `
<div
  className="App"
>
  <div
    className="bp3-navbar bp3-fixed-top App__Navbar bp3-dark"
  >
    <div
      className="bp3-navbar-group bp3-align-left"
    >
      <div
        className="bp3-navbar-heading App__Navbar__heading"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <button
            className="bp3-button bp3-minimal"
            disabled={undefined}
            onClick={undefined}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            tabIndex={undefined}
            type="button"
          >
            <span
              className="bp3-icon bp3-icon-search-around"
              icon="search-around"
            >
              <svg
                data-icon="search-around"
                fill={undefined}
                height={16}
                viewBox="0 0 16 16"
                width={16}
              >
                <desc>
                  search-around
                </desc>
                <path
                  d="M13.5 11c-.51 0-.98.15-1.38.42l-2.4-2.41c.17-.3.28-.64.28-1.01s-.11-.71-.28-1.01l2.41-2.41c.39.27.86.42 1.37.42a2.5 2.5 0 0 0 0-5A2.5 2.5 0 0 0 11 2.5c0 .51.15.98.42 1.38l-2.41 2.4C8.71 6.11 8.37 6 8 6s-.71.11-1.01.28l-2.41-2.4c.27-.4.42-.87.42-1.38a2.5 2.5 0 0 0-5 0A2.5 2.5 0 0 0 2.5 5c.51 0 .98-.15 1.38-.42l2.41 2.41C6.11 7.29 6 7.63 6 8s.11.71.28 1.01l-2.41 2.41c-.39-.27-.86-.42-1.37-.42a2.5 2.5 0 0 0 0 5A2.5 2.5 0 0 0 5 13.5c0-.51-.15-.98-.42-1.38l2.41-2.41c.3.18.64.29 1.01.29s.71-.11 1.01-.28l2.41 2.41c-.27.39-.42.86-.42 1.37a2.5 2.5 0 0 0 5 0 2.5 2.5 0 0 0-2.5-2.5zm0-10c.83 0 1.5.67 1.5 1.5S14.33 4 13.5 4 12 3.33 12 2.5 12.67 1 13.5 1zm-11 3C1.67 4 1 3.33 1 2.5S1.67 1 2.5 1 4 1.67 4 2.5 3.33 4 2.5 4zm0 11c-.83 0-1.5-.67-1.5-1.5S1.67 12 2.5 12s1.5.67 1.5 1.5S3.33 15 2.5 15zm11 0c-.83 0-1.5-.67-1.5-1.5s.67-1.5 1.5-1.5 1.5.67 1.5 1.5-.67 1.5-1.5 1.5z"
                  fillRule="evenodd"
                />
              </svg>
            </span>
            <span
              className="bp3-button-text"
            >
              HACKERBAY
            </span>
          </button>
        </a>
      </div>
      <div
        className="bp3-navbar-divider"
      />
      <div
        className="bp3-navbar-group bp3-align-left App__Navbar__leftmenu"
      >
        <button
          className="bp3-button bp3-minimal"
          disabled={undefined}
          onClick={undefined}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          tabIndex={undefined}
          type="button"
        >
          <span
            className="bp3-icon bp3-icon-projects"
            icon="projects"
          >
            <svg
              data-icon="projects"
              fill={undefined}
              height={16}
              viewBox="0 0 16 16"
              width={16}
            >
              <desc>
                projects
              </desc>
              <path
                d="M14 3c0-.55-.45-1-1-1H3c-.55 0-1 .45-1 1v1h12V3zm-2-3H4c-.55 0-1 .45-1 1h10c0-.55-.45-1-1-1zm3 5H1c-.55 0-1 .45-1 1v9c0 .55.45 1 1 1h14c.55 0 1-.45 1-1V6c0-.55-.45-1-1-1zm-3 6c0 .55-.45 1-1 1H5c-.55 0-1-.45-1-1V9h1v2h6V9h1v2z"
                fillRule="evenodd"
              />
            </svg>
          </span>
          <span
            className="bp3-button-text"
          >
            Products
          </span>
        </button>
        <button
          className="bp3-button bp3-minimal"
          disabled={undefined}
          onClick={undefined}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          tabIndex={undefined}
          type="button"
        >
          <span
            className="bp3-icon bp3-icon-dollar"
            icon="dollar"
          >
            <svg
              data-icon="dollar"
              fill={undefined}
              height={16}
              viewBox="0 0 16 16"
              width={16}
            >
              <desc>
                dollar
              </desc>
              <path
                d="M12.83 9.51c-.1-.3-.25-.58-.45-.84s-.45-.49-.75-.7c-.3-.2-.65-.36-1.05-.48-.16-.04-.43-.11-.8-.2-.35-.09-.73-.18-1.12-.28-.39-.1-.74-.19-1.06-.27-.31-.08-.49-.12-.54-.13-.43-.12-.78-.29-1.05-.52-.27-.23-.4-.55-.4-.95 0-.29.07-.53.21-.72.14-.19.32-.34.54-.46.22-.11.46-.19.72-.24.26-.05.52-.08.77-.08.74 0 1.35.15 1.83.46.48.3.75.83.81 1.56h2.14c0-.6-.13-1.13-.38-1.58-.25-.45-.59-.84-1.02-1.15-.43-.31-.93-.54-1.49-.7-.24-.06-.49-.1-.75-.14V1c0-.55-.45-1-1-1s-1 .45-1 1v1.08c-.23.03-.46.07-.68.13-.54.13-1.02.34-1.44.61-.42.28-.76.63-1.02 1.05-.26.43-.39.93-.39 1.5 0 .3.04.59.13.88.09.29.23.56.44.82.21.26.48.49.83.7.35.21.79.38 1.31.51.85.21 1.56.38 2.14.52.58.13 1.08.28 1.52.42.25.09.48.23.69.44.21.21.32.53.32.97 0 .21-.05.42-.14.63-.09.21-.24.39-.45.55-.21.16-.47.29-.81.39-.33.1-.73.15-1.2.15-.43 0-.84-.05-1.21-.14-.37-.09-.7-.24-.99-.43-.29-.2-.51-.45-.67-.76-.16-.31-.24-.68-.24-1.12H3c.01.71.15 1.32.43 1.84.27.52.64.94 1.1 1.27.46.33.99.58 1.61.74.27.07.56.12.85.16V15c0 .55.45 1 1 1s1-.45 1-1v-1.05c.3-.03.61-.08.9-.15.58-.13 1.1-.34 1.56-.63.46-.29.83-.66 1.11-1.11.28-.45.42-1 .42-1.64 0-.31-.05-.61-.15-.91z"
                fillRule="evenodd"
              />
            </svg>
          </span>
          <span
            className="bp3-button-text"
          >
            Pricing
          </span>
        </button>
        <button
          className="bp3-button bp3-minimal"
          disabled={undefined}
          onClick={undefined}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          tabIndex={undefined}
          type="button"
        >
          <span
            className="bp3-icon bp3-icon-dashboard"
            icon="dashboard"
          >
            <svg
              data-icon="dashboard"
              fill={undefined}
              height={16}
              viewBox="0 0 16 16"
              width={16}
            >
              <desc>
                dashboard
              </desc>
              <path
                d="M5 4c-.55 0-1 .45-1 1s.45 1 1 1 1-.45 1-1-.45-1-1-1zM4 7c-.55 0-1 .45-1 1s.45 1 1 1 1-.45 1-1-.45-1-1-1zm4-2c.55 0 1-.45 1-1s-.45-1-1-1-1 .45-1 1 .45 1 1 1zm-2 6c0 1.1.9 2 2 2s2-.9 2-2c0-.53-2-5-2-5s-2 4.47-2 5zM8 0C3.58 0 0 3.58 0 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6zm4-9c0-.55-.45-1-1-1s-1 .45-1 1 .45 1 1 1 1-.45 1-1zm0 2c-.55 0-1 .45-1 1s.45 1 1 1 1-.45 1-1-.45-1-1-1z"
                fillRule="evenodd"
              />
            </svg>
          </span>
          <span
            className="bp3-button-text"
          >
            Dashboard
          </span>
        </button>
        <button
          className="bp3-button bp3-minimal"
          disabled={undefined}
          onClick={undefined}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          tabIndex={undefined}
          type="button"
        >
          <span
            className="bp3-icon bp3-icon-compass"
            icon="compass"
          >
            <svg
              data-icon="compass"
              fill={undefined}
              height={16}
              viewBox="0 0 16 16"
              width={16}
            >
              <desc>
                compass
              </desc>
              <path
                d="M12 8c0 .14-.03.27-.08.39l-3 6.99c-.15.37-.51.62-.92.62s-.77-.25-.92-.61l-3-6.99a1.006 1.006 0 0 1 0-.79l3-6.99C7.23.25 7.59 0 8 0s.77.25.92.61l3 6.99c.05.13.08.26.08.4zM8 3.54L6.09 8h3.82L8 3.54z"
                fillRule="evenodd"
              />
            </svg>
          </span>
          <span
            className="bp3-button-text"
          >
            About Us
          </span>
        </button>
      </div>
    </div>
    <div
      className="bp3-navbar-group bp3-align-right App__Navbar__rightmenu"
    >
      <div>
        <a
          href="/signup"
          onClick={[Function]}
        >
          <button
            className="bp3-button bp3-minimal"
            disabled={undefined}
            onClick={undefined}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            tabIndex={undefined}
            type="button"
          >
            <span
              className="bp3-icon bp3-icon-badge"
              icon="badge"
            >
              <svg
                data-icon="badge"
                fill={undefined}
                height={16}
                viewBox="0 0 16 16"
                width={16}
              >
                <desc>
                  badge
                </desc>
                <path
                  d="M13.36 4.59c-.15-1.13.5-2.01 1.1-2.87L13.43.53c-1.72.88-4.12.65-5.63-.53-1.51 1.18-3.91 1.41-5.63.52l-1.03 1.2c.61.86 1.25 1.74 1.1 2.87-.3 2.29-2.45 4.17-1.32 6.68.45 1.14 1.44 1.9 2.72 2.2 1.56.36 3.52.72 4.16 2.53.64-1.81 2.6-2.16 4.16-2.54 1.28-.3 2.27-1.06 2.72-2.2 1.12-2.5-1.03-4.38-1.32-6.67z"
                  fillRule="evenodd"
                />
              </svg>
            </span>
            <span
              className="bp3-button-text"
            >
              Signup
            </span>
          </button>
        </a>
        <a
          href="/login"
          onClick={[Function]}
        >
          <button
            className="bp3-button bp3-minimal"
            disabled={undefined}
            onClick={undefined}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            tabIndex={undefined}
            type="button"
          >
            <span
              className="bp3-icon bp3-icon-log-in"
              icon="log-in"
            >
              <svg
                data-icon="log-in"
                fill={undefined}
                height={16}
                viewBox="0 0 16 16"
                width={16}
              >
                <desc>
                  log-in
                </desc>
                <path
                  d="M11 8c0-.28-.11-.53-.29-.71l-3-3a1.003 1.003 0 0 0-1.42 1.42L7.59 7H1c-.55 0-1 .45-1 1s.45 1 1 1h6.59L6.3 10.29c-.19.18-.3.43-.3.71a1.003 1.003 0 0 0 1.71.71l3-3c.18-.18.29-.43.29-.71zm4-8H9c-.55 0-1 .45-1 1s.45 1 1 1h5v12H9c-.55 0-1 .45-1 1s.45 1 1 1h6c.55 0 1-.45 1-1V1c0-.55-.45-1-1-1z"
                  fillRule="evenodd"
                />
              </svg>
            </span>
            <span
              className="bp3-button-text"
            >
              Login
            </span>
          </button>
        </a>
      </div>
    </div>
  </div>
  <div
    className="App__container"
  >
    <div
      className="HomePage"
    >
      <div>
        <h1>
          Hackerbay Server Monitoring
        </h1>
        <h2>
          Login / Signup To Get Started.
        </h2>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Example Intitial View 1`] = `
<div
  className="Example"
>
  <div>
    <h1>
      Example Component
    </h1>
    <h2>
      Name - 
      Sandun
    </h2>
    <h2>
      Age - 
      10
    </h2>
  </div>
</div>
`;

exports[`Storyshots Pages/HomePage Intitial View 1`] = `
<div
  className="HomePage"
>
  <div>
    <h1>
      Hackerbay Server Monitoring
    </h1>
    <h2>
      Login / Signup To Get Started.
    </h2>
  </div>
</div>
`;
